app/view/sdl/AlertManeuverPopUp.js

```diff
@@ -205,7 +201,7 @@ SDL.AlertManeuverPopUp = Em.ContainerView.create(
             SDL.AlertManeuverPopUp.iconsAreValid ? SUCCESS : WARNINGS,
             message.id,
             message.method,
-            SDL.AlertManeuverPopUp.iconsAreValid ? null : "Requested image(s) not found"
+            SDL.AlertManeuverPopUp.iconsAreValid ? null : (SDL.AlertManeuverPopUp.addErrorInfo ? "Requested image(s) not found" : null)
```

Comment:
@YarikMamykin maybe you can remove both `iconsAreValid` and `addErrorInfo` and just create a string variable and set error message which should be included?
```suggestion
            if (!checkImagesInRequest) { this.infoMessage = "Requested image(s) not found" }
            ...
            
            SDL.AlertManeuverPopUp.infoMessage
```

Reply:
@AKalinich-Luxoft take a look at https://github.com/LuxoftSDL/sdl_hmi/pull/9/commits/13aa30c58fb0a23262a6548fc80a8dabcb9cdb95